Student/SubmitAssignment/Assignments/AssignmentsViewController.swift

```diff
@@ -41,47 +41,34 @@ class AssignmentsViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        title = NSLocalizedString("Assignments", bundle: .core, comment: "")
-        tableView.dataSource = self
-        tableView.delegate = self
-        tableView.backgroundColor = .clear
-        view.backgroundColor = .clear
+        title = NSLocalizedString("Assignments", comment: "")
 
         activityIndicator.hidesWhenStopped = true
         addNavigationButton(UIBarButtonItem(customView: activityIndicator), side: .right)
 
-        assignments.refresh(force: true)
+        assignments.exhaust(force: true)
```

Comment:
It is already exhausting inside the useCase, but this makes it obvious that that happens, so the LoadingCell code is unnecessary.